src/test/kotlin/no/nav/syfo/oppfolgingstilfelle/bit/OppfolgingstilfelleBitSpek.kt

```diff
@@ -28,10 +21,8 @@ class OppfolgingstilfelleServiceSpek : Spek({
         )
 
         describe("Generate OppfolgingstilfelleList from OppfolgingsBitList") {
-            it("should return empty list of OppfolgingstilfelleBitList is empty") {
-                val result = oppfolgingstilfelleService.createoppfolgingstilfelleList(
-                    oppfolgingstilfelleBitList = emptyList(),
-                )
+            it("should return empty list if OppfolgingstilfelleBitList is empty") {
+                val result = emptyList<OppfolgingstilfelleBit>().generateOppfolgingstilfelleList()
```

Comment:
Synes disse testene burde bruke oppfolgingstilfelleService.createOppfolgingstilfellePerson sånn at den koden har ordentlige tester.

Reply:
Ja, jeg endret denne til OppfolgingstilfelleBitSpek siden dette egentlig bare var en unit-test for logikken i OppfolgingstilfelleBit.generateOppfolgingstilfelleList() (som ikke trenger ligge i service).

oppfolgingstilfelleService.createOppfolgingstilfellePerson blir testet via happy case i OppfolgingstilfelleApiSpek nå (API henter ut det som blir persistert fra kafka-prosesseringen), men kan være vi bør lage en egen OppfolgingstilfelleServiceSpek for å teste flere caser av oppfolgingstilfelleService.createOppfolgingstilfellePerson senere.